### ISSUE TITLE
l10n_ch_lsv_dd: unmet dependency l10n_ch_payment_slip_base_transaction_id for series 8.0

### ISSUE THREAD

**Issue by max3903** [OCA/l10n-switzerland#189]
The module l10n_ch_lsv_dd is part of the OCA Newsletter but it is not available on Odoo Apps.

Here is why:

```
l10n_ch_payment_slip_base_transaction_id: Module not installable
l10n_ch_payment_slip_account_statement_base_completion: Module not installable
l10n_ch_dta_base_transaction_id: Module not installable
l10n_ch_lsv_dd: unmet dependency l10n_ch_payment_slip_base_transaction_id for series 8.0
```

**Comment by yvaucher** [comment]
Ok my merge deleted that change when rebasing.

It was activated in #162 I'll fix it

**Comment by yvaucher** [comment]
Fixed at 5102863
